heroes.html

```diff
@@ -20,7 +20,7 @@
         </a>
         <ul>
           <li>
-            <a href="heroes.html">Heroes</a>
+            <a>Heroes</a>
```

Comment:
Href is needed, please add

Reply:
@sbldwn sorry about that! I went ahead and added it back. I appreciate it!